app/models/edition/audit_trail.rb

```diff
@@ -28,17 +28,34 @@ def self.acting_as(actor)
   end
 
   def record_create
-    versions.create event: 'create', user: Edition::AuditTrail.whodunnit, state: state
+    user = Edition::AuditTrail.whodunnit
+    versions.create event: 'create', user: user, state: state
+    alert!(user)
   end
   private :record_create
 
   def record_update
     if changed.any?
-      versions.build event: 'update', user: Edition::AuditTrail.whodunnit, state: state
+      user = Edition::AuditTrail.whodunnit
+      versions.build event: 'update', user: user, state: state
+      alert!(user)
     end
   end
   private :record_update
 
+  def alert!(user)
+    if should_alert_for?(user)
+      Notifications.edition_published_by_monitored_user(user).deliver_now
+    end
+  end
+  private :alert!
+
+  def should_alert_for?(user)
+    ENV['CO_NSS_WATCHKEEPER_EMAIL_ADDRESS'].present? &&
```

Comment:
Do we need the check for `present?` if we're checking it's equal to the email address?

Reply:
Nope, equality is good. I'll also only add this env var on production so we don't send emails for other environments.